test/extensions/filters/network/ratelimit/config_test.cc

```diff
@@ -96,44 +62,15 @@ TEST(RateLimitFilterConfigTest, RatelimitCorrectProto) {
   cb(connection);
 }
 
-TEST(RateLimitFilterConfigTest, RatelimitEmptyProto) {
-  std::string json_string = R"EOF(
```

Comment:
Not sure why we are merging some config here with empty proto config and testing. I thought it is better to test with empty proto - this may not be created. If it is not, LMK, I will correct the test case accordingly